### ISSUE TITLE
jscpd file: TypeError: Cannot destructure property 'path' of 'entry' as it is undefined

### ISSUE THREAD

**Issue by emanuelb** [kucherenko/jscpd#383]
Running `jscpd file_that_exists` (such as /etc/passwd) return:

```
~/node_modules/jscpd/bin $ ./jscpd /etc/passwd
TypeError: Cannot destructure property 'path' of 'entry' as it is undefined.
    at detect (/home/appuser/node_modules/@jscpd/finder/dist/in-files-detector.js:42:21)
    at InFilesDetector.detect (/home/appuser/node_modules/@jscpd/finder/dist/in-files-detector.js:66:16)
    at Object.jscpd (/home/appuser/node_modules/jscpd/dist/index.js:46:25)
    at Object.<anonymous> (/home/appuser/node_modules/jscpd/bin/jscpd:15:12)
    at Module._compile (internal/modules/cjs/loader.js:1256:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1277:10)
    at Module.load (internal/modules/cjs/loader.js:1105:32)
    at Function.Module._load (internal/modules/cjs/loader.js:967:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:60:12)
    at internal/main/run_main_module.js:17:47
```

**Comment by kucherenko** [comment]
Thank you for the report, will fix

**Comment by kucherenko** [comment]
fixed at jscpd@3.3.3
